### PR TITLE
Add Pricing App

### PR DESCRIPTION
**Pull request by nmisek**
Adds an AMPL HiGHS version of Gurobi avocado price optimization model: 

https://colab.research.google.com/github/Gurobi/modeling-examples/blob/master/price_optimization/price_optimization.ipynb

### CODE REVIEW

File: .nextmv/golden/price-optimization-ampl/ampl_model.mod
Comment (sebastian-quintero) [inline .nextmv/golden/price-optimization-ampl/ampl_model.mod:1]: I don’t think this `.mod` file is supposed to be in this location, right?

File: price-optimization-ampl/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+v1.3.2
```
Comment (sebastian-quintero) [inline price-optimization-ampl/VERSION]:
```suggestion
v0.0.0
```

Set an initial version which will be updated upon release.

File: price-optimization-ampl/input.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "coefficients": {
+    "region": [
+      0, -0.242693153677881, 1.433019836643242, -3.019243790637881,
+      1.7138120739734646, 0.5839901666611692, 2.6017994486412657,
+      -1.8150328723904097
+    ],
+    "price": -2.2037701048902467,
+    "intercept": 5.43931005216503,
+    "year_index": 0.16076930231844555,
+    "peak": 0.548510505830839
```
Comment (sebastian-quintero) [inline price-optimization-ampl/input.json:35]: I think we should make an effort to to use a reasonable precision for these numbers, where it applies. Maybe the coefficients for the regression have a lot of decimals but the minimum and maximum product allocation I would’ve expected to see some integers there.

**Review by sebastian-quintero — CHANGES_REQUESTED** [review]
The version comment is blocking. The rest of the comments are non-blocking.

File: .nextmv/golden/price-optimization-ampl/ampl_model.mod
Comment (nmisek) [reply .nextmv/golden/price-optimization-ampl/ampl_model.mod:1]: fixed by adding an arg for folder containing mod file

File: price-optimization-ampl/input.json (hunk 2 above)
Comment (nmisek) [reply price-optimization-ampl/input.json:35]: fixed by rounding and adding a note to readme

**Review by sebastian-quintero — APPROVED** [review]
🌮 🌮 🌮 🌮 🌮

**Pull request merged**